client/src/pages/sessions/session.tsx

```diff
@@ -81,24 +93,37 @@ const SessionTable = ({ search }: { search: any }) => {
     };
   }, []);
 
+  function canEdit() {
```

Comment:
reuse imported (will be a big help when, say, changing `isAdmin`+`isContentManager` to `role`)